src/state.cuh

```diff
@@ -2152,8 +2188,11 @@ public:
                 );
                 _content->touch = tmp_touch;
                 _content->touch[account_idx] = 0;
-                delete dup_account;
             )
+
+            delete dup_account;
+            dup_account = nullptr;
```

Comment:
Shared memory must be free just by one thread if allocated by one thread. I will fix it.